frameworks/AutoGluon/exec.py

```diff
@@ -57,17 +60,8 @@ def run(dataset, config):
     with Timer() as predict:
         predictions = predictor.predict(X_test)
 
-    probabilities = predictor.predict_proba(X_test, as_pandas=True) if is_classification else None
-    if is_classification and len(probabilities.shape) == 1:
-        # for binary, AutoGluon returns only one probability and it's not systematically the smaller or greater label.
-        # Here, we're trying to identify to which label correspond the probabilities
-        # and from there we can build the probabilities for all labels.
-        classes = dataset.target.classes
-        prob_class = next((predictions[i] if p > 0.5 else next(c for c in classes if c != predictions[i])
-                           for i, p in enumerate(probabilities) if p != 0.5),
-                          classes[1])
-        probabilities = (np.array([[1-row, row] for row in probabilities]) if prob_class == classes[1]
-                         else np.array([[row, 1-row] for row in probabilities]))
+    probabilities = predictor.predict_proba(dataset=X_test, as_pandas=True, as_multiclass=True) if is_classification else None
+    prob_labels = probabilities.columns.values.tolist()
```

Comment:
This will break if is_classification == False, since probabilities will be None

Reply:
good point, I should enable the regression dataset in the test benchmark...